### PR TITLE
[MID-27] add feedback open,send and close buttton

### PR DESCRIPTION
**Pull request by BigTako**
Added buttons for header(open feedback modal), feedback modal(send feedback) and close feedback modal. [Ticket](https://trello.com/c/v29viogj/27-%D0%B7%D0%B2%D0%BE%D1%80%D0%BE%D1%82%D0%BD%D1%96%D0%B9-%D0%B7%D0%B2%D1%8F%D0%B7%D0%BE%D0%BA-%D0%BA%D0%BE%D0%BC%D0%BF%D0%BE%D0%BD%D0%B5%D0%BD%D1%82-%D0%BA%D0%BD%D0%BE%D0%BF%D0%BA%D0%B0) 
Deploy: https://feedback-button-demo-bigtako-bigtakos-projects.vercel.app?_vercel_share=qozEI66RJ23CAGnPmqRHQONWtP3t3Xcp

### CODE REVIEW

**Review by Yivgha — APPROVED** [review]
Magnificent!

File: src/app/_components/CrossIcon/CrossIcon.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+export default function PlusIcon() {
```
Comment (alex-popov-tech) [inline src/app/_components/CrossIcon/CrossIcon.js]: 🟢  can you please replace default export with named?

File: src/app/_components/CrossIcon/index.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export { default } from './CrossIcon';
```
Comment (alex-popov-tech) [inline src/app/_components/CrossIcon/index.js]: 🔴 please stop using 2 files and 1 folder for each component, there is no point in doing so

File: src/app/utils/cn.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import { clsx } from 'clsx';
+import { configuredTwMerge } from './configuredTwMerge';
+
+export default function cn(...inputs) {
```
Comment (alex-popov-tech) [inline src/app/utils/cn.js]: 🔴 please do not use default export, that will allow to import it like `import { cn } from '@/app/utils'`

Comment (alex-popov-tech) [reply src/app/utils/cn.js]: 🟠  and meaningful name like `classes` would be nice here

File: src/app/styles/constants/buttonCoreStyles.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
+import { stylesCore } from './stylesCore';
+
+export const buttonColorVariant = {
```
Comment (alex-popov-tech) [inline src/app/styles/constants/buttonCoreStyles.js]: 🟠 aren't `Button` related styles are suppose to be near `Button` component?

**Review by alex-popov-tech — CHANGES_REQUESTED** [review]

File: src/app/_components/CrossIcon/index.js (hunk 2 above)
Comment (BigTako) [reply src/app/_components/CrossIcon/index.js]: Thanks for comment! Done.

File: src/app/utils/cn.js (hunk 3 above)
Comment (BigTako) [reply src/app/utils/cn.js]: Thanks for comment! Done.

File: src/app/styles/constants/buttonCoreStyles.js (hunk 4 above)
Comment (BigTako) [reply src/app/styles/constants/buttonCoreStyles.js]: Thanks for comment! Done.

File: src/app/_components/CrossIcon/CrossIcon.js (hunk 1 above)
Comment (BigTako) [reply src/app/_components/CrossIcon/CrossIcon.js]: Thanks for comment! Done.

File: docker-compose.yml [hunk 5]
```diff
@@ -1,4 +1,4 @@
-version: '3.8'
+version: "3.8"
```
Comment (alex-popov-tech) [inline docker-compose.yml:1]: 🟠 sholdn't that be single quote?

File: .github/workflows/linting.yaml [hunk 6]
```diff
@@ -5,7 +5,7 @@ name: linting
 on:
   push:
     branches:
-      - '*'
+      - "*"
```
Comment (alex-popov-tech) [inline .github/workflows/linting.yaml:8]: 🟠 shouldn't that be single quote?

File: src/app/_components/PillButton/PillButton.js [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+    active,
+    disabledState,
+    className,
+  );
+
+  return (
+    <button className={styles} disabled={disabled} onClick={onClick}>
+      <div className={layoutStyle || ''}>
+        {icon}
+        <Paragraph className={'text-inherit'}>{children}</Paragraph>
```
Comment (alex-popov-tech) [inline src/app/_components/PillButton/PillButton.js]: 🟢 i think there are extra {} around 'text-inherit'

File: src/app/_components/PillButton/pillButtonStyles.js [hunk 8]
```diff
@@ -0,0 +1,64 @@
+import { stylesCore } from '@/app/styles/constants/stylesCore';
```
Comment (alex-popov-tech) [inline src/app/_components/PillButton/pillButtonStyles.js]: 🟠 `pillButtonStyles.js` => `styles.js`

File: src/app/_components/PillButton/PillButton.js [hunk 9]
```diff
@@ -0,0 +1,70 @@
+import React from 'react';
```
Comment (alex-popov-tech) [inline src/app/_components/PillButton/PillButton.js]: 🟠 `PillButton` => `component` ? so structure will be
`PillButton/component.js`
`PillButton/styles.js`

File: src/app/styles/constants/stylesCore.js [hunk 10]
```diff
@@ -0,0 +1,6 @@
+export const stylesCore = {
```
Comment (alex-popov-tech) [inline src/app/styles/constants/stylesCore.js]: 🟢 can't these few styles be placed in button styles? as plain consts

File: docker-compose.yml (hunk 5 above)
Comment (BigTako) [reply docker-compose.yml:1]: Discussed in private conversation.

File: .github/workflows/linting.yaml (hunk 6 above)
Comment (BigTako) [reply .github/workflows/linting.yaml:8]: Discussed in private conversation.

File: src/app/_components/PillButton/pillButtonStyles.js (hunk 8 above)
Comment (BigTako) [reply src/app/_components/PillButton/pillButtonStyles.js]: Thanks for comment! Done.

File: src/app/_components/PillButton/PillButton.js (hunk 7 above)
Comment (BigTako) [reply src/app/_components/PillButton/PillButton.js]: Thanks for comment! Done.

File: src/app/_components/PillButton/PillButton.js (hunk 9 above)
Comment (BigTako) [reply src/app/_components/PillButton/PillButton.js]: Thanks for comment! Done.

File: src/app/styles/constants/stylesCore.js (hunk 10 above)
Comment (BigTako) [reply src/app/styles/constants/stylesCore.js]: Thanks for comment! This functionality is useless for now, so i completely removed it.

**Review by alex-popov-tech — APPROVED** [review]

**Review by yataras — APPROVED** [review]

**Pull request merged**